pkg/api/validation/validation.go

```diff
@@ -3198,7 +3198,7 @@ func ValidateLoadBalancerStatus(status *api.LoadBalancerStatus, fldPath *field.P
 			}
 		}
 		if len(ingress.Hostname) > 0 {
-			for _, msg := range validation.IsDNS1123Subdomain(ingress.Hostname) {
+			for _, msg := range validation.IsDNS4592WildcardSubdomain(ingress.Hostname) {
```

Comment:
Under what conditions would we have a wildcard in the LB status field?  This is a breaking change - the semantic is that this field can be used as a network endpoint.  Wildcard would break this.